main.py

```diff
@@ -220,11 +220,15 @@ def vms(self):
         vms = []
 
         for line in out.splitlines():
-            m = re.search(r'"([a-zA-Z0-9-_]+)"', line)
-            if m:
-                name = m.group(1)
+            m = re.findall(r'([a-zA-Z0-9-_]+)', line)
+            if m[0] != "inaccessible":
```

Comment:
If you use the same string several times, you may consider taking it out into a constant (of course only if you think it is more readable - up to you).

Reply:
Your right (generaly), but in this case it will look like:
inaccessible = "inaccessible"
bla bla != inaccessible 
then bla bla = inaccessible 

which only add more line to the function. 
for now I dont use this in any other places...